bundles/org.eclipse.swt/META-INF/p2.inf

```diff
@@ -44,3 +44,13 @@ requires.9.namespace = org.eclipse.equinox.p2.iu
 requires.9.name = org.eclipse.swt.gtk.linux.riscv64
 requires.9.range = [$version$,$version$]
 requires.9.filter = (&(osgi.os=linux)(osgi.ws=gtk)(osgi.arch=riscv64)(!(org.eclipse.swt.buildtime=true)))
+
```

Comment:
This part can not go in unless actual binaries are produced and pushed .

Reply:
Not only this specific file, but without the SWT binaries being officially compiled for FreeBSD, this whole PR is moot/pointless and shouldn't be merged at all.

And with the SWT being a critical component of Eclipse, failure to merge this PR is also a critical failure for the whole effort in eclipse-platform/eclipse.platform.releng.aggregator#2959